server/helm/rayyan-store/templates/secrets/rayyan-store.yaml

```
@@ -0,0 +1,10 @@
+{{- range $index, $url := .Values.secrets.RAYYAN_STORE_DATABASE_URLS }}
```

Now you can de-scope the secret name (no more monolith chart):
```suggestion
{{- range $index, $url := .Values.secrets.DATABASE_URLS }}
```